### PR TITLE
midi for light controller

### PR DESCRIPTION
**Pull request by dg3nec**
A controller with many midi outputs. Beat, vu-meter, timecode,
deckchanges, bpm.  For better light and  be in sync to the musik. Works
fully automated to and with the current deck and has no problem with
pitch. Talked (mailed) about this with daniel, he said make a pull in
git. I think to force communication with light software is a good
feature for mixxx ;-) . (write wiki page is next step).

### CODE REVIEW

**Comment by dg3nec** [comment]
yes safe to test

File: res/controllers/Midi_for_light.midi.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MixxxMIDIPreset mixxxVersion="1.12+" schemaVersion="1">
+	<info>
+		<name>MIDI for light</name>
+		<author>Michael Stahl / DG3NEC</author>
+		<description>MIDI-output for light control</description>
+		<forums>http://www.mixxx.org/forums</forums>
```
Comment (rryan) [inline res/controllers/Midi_for_light.midi.xml]: Could you create a stub forum post to link to here? (just post in the controllers section with an empty post with the title of your preset)

**Comment by dg3nec** [comment]
OK, i have opened a thread in forum - controller

**Comment by sblaisot** [comment]
Please add new files to Windows nsis installer. 

Thanks.

**Comment by Be-ing** [comment]
Please document how to use this on the wiki.

**Comment by dg3nec** [comment]
In Wiki is a "Tutorial" how to use. Reason bacuse Tutorial: It is realised like a controller, but there is no real controller. The midi information are send normaly to a light software like PCDimmer, QLC or what ever. Internal or by network.
Looka at: http://mixxx.org/wiki/doku.php/midi_for_light

**Comment by dg3nec** [comment]
nsis install are updated with pull repuest "Adding controller Midi_for_light #820"

**Comment by dg3nec** [comment]
now a correct forum link and wiki link is inserted in xml file

**Comment by rryan** [comment]
Superseded by #824

**Pull request closed without merging**